### PR TITLE
Alec ReadMe Branch

### PR DESCRIPTION
**Pull request by AlecFaye**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+❌ - 👥 Role Models  
+
+✔️ - Completed  
+🚧 - In Progress  
+❌ - Not Started  
+
+### Languages and Tools:
+Sachbox is a project to learn front-end and back-end development, art, and version control.
+
+<img align="left" alt="Visual Studio Code" width="30px" src="https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/visual-studio-code/visual-studio-code.png" />
```
Comment (SachJay) [inline README.md:24]: This shit is poggggg

File: README.md [hunk 2]
```diff
@@ -1 +1,44 @@
-# SachBoxNew
\ No newline at end of file
+# SachBox
+
+## Description
+SachBox is an expansive game that connects communities who have been disconnected due to the recent pandemic through online party games.
+
```
Comment (SachJay) [inline README.md:5]: Can we also add a section on how to download and start up the project. 

Download
1) git clone 
2) npm install

Run
1) npm start
2) node server.js (in progress)

File: README.md [hunk 3]
```diff
@@ -1 +1,44 @@
-# SachBoxNew
\ No newline at end of file
+# SachBox
+
+## Description
+SachBox is an expansive game that connects communities who have been disconnected due to the recent pandemic through online party games.
```
Comment (SachJay) [inline README.md:4]: 👌 we really should figure out a legit name guess thats after

also the links at the bottom is sickkkk

**Review by SachJay — CHANGES_REQUESTED** [review]

**Comment by AndCrit** [comment]
Looks good Bosmang. One thing since we plan on adding more of the features then the README won't be considered done until we finish those things once the eventual first merge goes through

File: README.md (hunk 2 above)
Comment (AlecFaye) [reply README.md:5]: Added the project instructions - will fix it up

Comment (SachJay) [reply README.md:5]: my bad, can we be more detailed looool
in the Git doc there is a section on `npm start` and `git clone`

for `node server.js` have a note saying you have to be in "/my-app"

**Review by LukeBacchus — APPROVED** [review]
Good

File: README.md (hunk 2 above)
Comment (AlecFaye) [reply README.md:5]: Added more info, I do not know if that is what you wanted. If not, tell me and I'll update it.

Comment (AlecFaye) [reply README.md:5]: Git DOC NVM IGNORE ME, I AM BRAIN DEAD

Comment (AlecFaye) [reply README.md:5]: Added the info from the Git doc

**Review by SachJay — APPROVED** [review]
perfecttt thanks

**Comment by SachJay** [comment]
when merging put the merge settings to "Squash and Merge"
so there's only 1 commit in master

image

**Comment by SachJay** [comment]
@LukeBacchus xd let the person who owns the ticket press the merge button
they might have found something that was a problem and it just feels good to press it yourself when you finish the ticket

But anywaysss nice Kelvin man got the first merge in

**Comment by AlecFaye** [comment]
Lol its alright, hell yeah!

**Pull request merged**